### ISSUE TITLE
Add `SAVE` button to save plot in GUI mode.

### ISSUE THREAD

**Issue by aditodkar** [NITDgpOS/PlotIt#81]
Currently the GUI mode does not have any button to `SAVE` plot.

![screenshot_1](https://user-images.githubusercontent.com/16613832/34461034-95bc505e-ee45-11e7-9eab-bd2d176b35b2.png)

**Comment by realsdx** [comment]
I would like to work on this issue

**Comment by djmgit** [comment]
@realsdx please go ahead :)

**Comment by realsdx** [comment]
Done.. should i submit a PR ? as my previous PR still not merged.. but i already merged all the branches in my fork

**Comment by realsdx** [comment]
Yes, I have used a different branch... but it also has the quitting problem solved. I need to remove the commit and rebase it as you cannot reproduce it for now.
